lib/class-wp-rest-menu-items-controller.php

```diff
@@ -618,7 +608,7 @@ public function get_item_schema() {
 
 		$schema['properties']['type_label'] = array(
 			'description' => __( 'The singular label used to describe this type of menu item.' ),
-			'context'     => array( 'view' ),
+			'context'     => array( 'view', 'embed' ),
```

Comment:
The `edit` context should be added here. `edit` is almost (always?) a superset of the other contexts in core.

Reply:
This is an editable field. `type_label` is generated for object type.

Reply:
It doesn't really matter. Whether the field is editable or not is marked by the `readonly` field. By excluding the field from the `edit` context, a user would have to make two API requests to get that information. When typically, by requesting the resource in the `edit` context ( the context that requires the highest capabilities ), they'd retrieve all fields.